main.tex

```diff
@@ -159,6 +159,17 @@ \subsection{Terminology and notation}
 
 Finally, $k$ is a protocol level constant, denoting the number of credentials used in input and output registration requests, and $v_{\mathit{max}} = 2^{51}-1$ constrains the amount value ranges.\footnote{$\log_2(2099999997690000) \approx 50.9$}
 
+\subsection{Scheduling}
+
+A CoinJoin round consists of an Input Registration, an Output Registration and a Transaction Signing phases. The goal of WabiSabi is to ensure the inputs of users those do not comply with the protocol are identified, thus these inputs can be excluded from the following rounds in order to ensure completion of the protocol. Discussion of various blame strategies, methods to avoid timing attacks and other scheduling related topics are not part of this overview, rather we concentrate on identifying non-compliant inputs in various phases of the protocol:
```

Comment:
```suggestion
A CoinJoin round consists of an Input Registration, an Output Registration and a Transaction Signing phases. The goal of WabiSabi is to ensure the inputs of users who do not comply with the protocol are identified, thus these inputs can be excluded from the following rounds in order to ensure completion of the protocol. Discussion of various blame strategies, methods to avoid timing attacks and other scheduling related topics are not part of this overview. Rather we concentrate on identifying non-compliant inputs in various phases of the protocol:
```

Reply:
I can't spot the diff, but I've read your suggestion and it is correct, so I ACK it.